docs/querying/sql.md

```diff
@@ -313,48 +313,51 @@ possible for two aggregators in the same SQL query to have different filters.
 
 Only the COUNT and ARRAY_AGG aggregations can accept DISTINCT.
 
+When no rows are selected, aggregate functions will return their initialized value for the grouping they belong to. What this value is exactly for a given aggregator is dependent on the configuration of Druid's SQL compatible null handling mode, controlled by `druid.generic.useDefaultValueForNull`. The table below defines the initial values for all aggregate functions in both modes.
```

Comment:
Hmm this paragraph reads oddly to me for two reasons:

- At first blush II think a typical user would think it's impossible for groups to exist that do not have any rows. (It isn't typical SQLy behavior.) So we should list some examples of when the default value will show up. I can think of two cases: grand total (aggregations with no `group by`) and filtered aggregators where the filter does not match any rows within the group.
- Not all aggregators have behavior dependent on `druid.generic.useDefaultValueForNull`, so it's not technically correct to say this categorically. I don't think we need to mention `druid.generic.useDefaultValueForNull` at all here, actually, because the individual aggregators in the table below call it out when appropriate. Or, if we do mention it, we could just say that it "may depend on" rather than "is dependent on".

Welcome to opinions from others about how to express this most clearly.